Test/MonoGame.Tests.XNA.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MonoGame.Tests</RootNamespace>
     <AssemblyName>MonoGame.Tests.XNA</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
I am not sure this works. 
2010 is the last VS that is officially supported by XNA and VS2010 can work with .Net framework up to 4.0.

Reply:
I would love to know the market share for things like VS2010/12/13 etc. It would be nice to know when we can drop support for those IDE's.

Reply:
This is about the XNA tests. 
MG dropped end-user support of VS2010 years ago when upgrading from .net 4.0 to 4.5.

Reply:
I've been using this to run XNA on VS2023/15...

https://mxa.codeplex.com/releases

It looks weird, but works really really well.

The reason for the change is we have some .NET 4.5 only code in the unit tests now.  So it was either remove that code or update the project.

Reply:
Ok , I see. I would bet you had one of those.